src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -634,6 +653,28 @@ void cannotCreateTlrWhenUserAlreadyRequestedAnItemFromTheSameTitle() {
       hasParameter("instanceId", item1.getInstanceId().toString()))));
   }
 
+  @ParameterizedTest
+  @CsvSource({"Hold", "Recall"})
+  void cannotCreateHoldTlrWhenAvailableItemForInstance(String requestType) {
+    configurationsFixture.enableTlrFeature();
+
+    List<ItemResource> items = itemsFixture.createMultipleItemsForTheSameInstance(2);
+    ItemResource item1 = items.get(0);
```

Comment:
If you're only using one item, maybe it can be named just "item"?

Reply:
renamed